daemons/gptp/linux/src/linux_hal_persist_file.cpp

```diff
@@ -123,7 +123,9 @@ class LinuxGPTPPersistFile : public GPTPPersist {
 
 		bool result = false;
 		if (memoryDataLength > storedDataLength) {
-			ftruncate(persistFD, memoryDataLength);
+			int ret = ftruncate(persistFD, memoryDataLength);
```

Comment:
Thanks for your comment @pinealservo. I have added a new commit to print a message when a non-success code is returned.

Regards,
Roland Hii